src/main/java/me/zeroeightsix/kami/module/modules/movement/ElytraFlight.kt

```diff
@@ -389,21 +421,22 @@ class ElytraFlight : Module() {
         } else downSpeedControl.value.toDouble()
 
         /* Hover */
-        if (hoverTarget < 0.0 || moveUp) hoverTarget = mc.player.posY else if (moveDown) hoverTarget = mc.player.posY - calcDownSpeed
-        hoverState = (if (hoverState) mc.player.posY < hoverTarget else mc.player.posY < hoverTarget - 0.1) && altitudeHoldControl.value
+        if (!pathfind) {
+            if (hoverTarget < 0.0 || moveUp) hoverTarget = mc.player.posY else if (moveDown) hoverTarget = mc.player.posY - calcDownSpeed
+        }
```

Comment:
Why is this disabled when path finding?

Reply:
we need the decimal part of the y level to be between  .15 and  .35  so we only need to evade blocks on that y level instead of the one above/bleow too.  you could enable if if you make sure the player always ends up between .1 and  .35.
you'd have to force a new path then too if you change the y level. (Astarpathfinder l. 359 check if the y level changed) Further comment on the calcPath method: i am forcing a new path every time the current one is shorter than 20 blocks because sometimes there is a good path but the chunks are not loaded yet -> it is better to force a new one once you are closer rather than continuing a potentially suboptimal path (suboptimal due to missing chunks)

Reply:
makes enough sense. ideally I would have y level changing supported